### PR TITLE
6: Deactivate projects.

### PR DESCRIPTION
**Pull request by robherba**
**Steps to test**
- Create a new project node.
- Check if the new VBO action is executed successfully.

**DoD**
- In projects view (created in a previous ticket) ensure we have a VBO action to enable/disable projects.

[issue](https://github.com/ManatiCR/manatime/issues/6)

### CODE REVIEW

File: modules/features/manatime_feature_projects/manatime_feature_projects.module [hunk 1]
```diff
@@ -17,3 +17,41 @@ function manatime_feature_projects_permission() {
...
 }
+
+/**
+ * Implements hook_action_info().
+ */
+function manatime_feature_projects_action_info() {
+  return array(
+    'manatime_feature_projects_action_enable' => array(
+      'type' => 'entity',
+      'label' => t('Enable projects'),
```
Comment (kporras07) [inline modules/features/manatime_feature_projects/manatime_feature_projects.module]: Activate Projects

File: modules/features/manatime_feature_projects/manatime_feature_projects.module [hunk 2]
```diff
@@ -17,3 +17,41 @@ function manatime_feature_projects_permission() {
...
+      'behavior' => array('changes_property'),
+      'configurable' => FALSE,
+      'vbo_configurable' => FALSE,
+      'triggers' => array('any'),
+    ),
+    'manatime_feature_projects_action_disable' => array(
+      'type' => 'entity',
+      'label' => t('Disable projects'),
```
Comment (kporras07) [inline modules/features/manatime_feature_projects/manatime_feature_projects.module]: Deactivate Projects

**Comment by robherba** [comment]
Ok, changes has been applied, please check! @kporras07

**Comment by kporras07** [comment]
Assuming Wercker passes, :shipit:

**Pull request merged**